app/services/manual/queue_publish_service.rb

```diff
@@ -12,6 +12,12 @@ def call
         manual_id: manual.id,
         version_number: manual.version_number,
       )
+
+      govuk_header_params = {
+        request_id: GdsApi::GovukHeaders.headers[:govuk_request_id],
+        authenticated_user: GdsApi::GovukHeaders.headers[:x_govuk_authenticated_user],
+      }
+
```

Comment:
Out-of-scope: I think it would be better to extract these header values within the controller action and pass them into the service as constructor args.